index.html

```diff
@@ -30,7 +31,7 @@
         <li><a href="#contact-section" class="menu-link">Contact</a></li>
       </ul>
     </div>
-    <main>
+    <main class="stop">
```

Comment:
https://github.com/khitermedachraf/my-professional-portfolio/blob/228ec5fa9bb78f4710358691ecea5f5522bf3229/index.html#L92-L187

- [ ] In this particular project, the whole projects section along with the modals should be created dynamically with Javascript.

The base projects section is already done (good job!), but the modals are still not being created with Javascript.  Kindly please use Javascript to dynamically render your modals.

You could reuse the code that you already have.

That way, adding projects in the future will be easy. You will only need to add it in one place - as another object in the `data` array.

![image](https://user-images.githubusercontent.com/57421823/152509546-f41b40b6-dc88-4e3f-9ab4-7bee886e1f52.png)

- Also, you could put some id on all the projects, and make sure the id of each project matches the id of the said project on the data array. Then, iteratively add an event listener to each button calling the correct id, so that the correct project id is shown when pressed. (These are just some ideas I hope it's helpful).